crypto/lhash/lh_stats.c

```diff
@@ -61,17 +61,21 @@ void OPENSSL_LH_node_usage_stats(const OPENSSL_LHASH *lh, FILE *fp)
 
 void OPENSSL_LH_stats_bio(const OPENSSL_LHASH *lh, BIO *out)
 {
+#ifndef OPENSSL_NO_STATS
     int omit_tsan = 0;
 
-#ifdef TSAN_REQUIRES_LOCKING
+# ifdef TSAN_REQUIRES_LOCKING
     if (!CRYPTO_THREAD_read_lock(lh->tsan_lock)) {
         BIO_printf(out, "unable to lock table, omitting TSAN counters\n");
         omit_tsan = 1;
     }
+# endif
 #endif
+
     BIO_printf(out, "num_items             = %lu\n", lh->num_items);
     BIO_printf(out, "num_nodes             = %u\n",  lh->num_nodes);
     BIO_printf(out, "num_alloc_nodes       = %u\n",  lh->num_alloc_nodes);
+#ifndef OPENSSL_NO_STATS
```

Comment:
Personally, I'd remove this #ifdef and the prior #endif and have the function output _nothing_ in a no-stats build.

The same could apply to the other functions here because they are all about statistics presentation.